README.md

```diff
@@ -81,18 +82,14 @@ main(
 
 ## Prompts
 
-Prompts live in `data/prompts/user/` as markdown files and may include placeholders such as:
+Prompts may include placeholders such as:
 
 * `{context}`
 * `{file_contents}`
+* `{file_references}`
```

Comment:
This is interesting, I don't think the code actually supports this placeholder. Revert this change, and please make a separate PR to update the existing prompts to remove this placeholder. (In general, prefer making separate PRs to adding things onto an existing PR.)